### DISCUSSION TITLE
Simple UI Change

### DISCUSSION THREAD

**Discussion by stccorp** [plausible/analytics#574] [Anything Else]
Hello,

I just created and account and had to open a request on github because I could not find where to add more websites.  The solution was "simple", click on the plausible logo.

A normal user will expect the Plausible logo to take you to the Plausible homepage.

My suggestion is, just make a link that says "DashBoard" above the settings that take you to the top, instead of using the Plausible logo

![brave_2021-01-09_14-03-50](https://user-images.githubusercontent.com/8180135/104106460-07813100-5284-11eb-8bac-e9352785d586.png)

**Comment by stccorp** [comment]
a better option, Having an "All" link
![brave_2021-01-10_02-15-56](https://user-images.githubusercontent.com/8180135/104132167-812f2280-5349-11eb-9a69-34f24126fb41.png)

**Comment by KristerV** [comment]
hmm. I have not gotten stuck on this. But that's probably because I assume the main logo will go to the main page of what I'm interested in. Some websites go on their home marketing page and that always frustrates me. So Plausible in my eyes is doing it right.

But this just proves there is no single understanding about what the main logo does, so if I was the plausible team I'd try fix it :D

**Comment by ukutaht** [comment]
Could also show a 'Add site' button in the dropdown that lists all the sites.

**Reply by stccorp** [reply]
I like that idea
